src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/SendToBulkPrintHandler.java

```diff
@@ -138,32 +138,49 @@ public DispatchPriority getPriority() {
     }
 
     private void updateCaseToFlagError(SscsCaseData caseData, String description) {
-        caseData.setHmctsDwpState("failedSending");
-        ccdService.updateCase(caseData,
-            Long.valueOf(caseData.getCcdCaseId()),
-            EventType.SENT_TO_DWP_ERROR.getCcdType(),
-            "Send to FTA Error",
-            description,
-            idamService.getIdamTokens());
+        Long caseId = Long.valueOf(caseData.getCcdCaseId());
+
+        log.info("About to update case v2 with send to bulk print event - error for id {}", caseId);
+        updateCcdCaseService.updateCaseV2(
+                caseId,
+                EventType.SENT_TO_DWP_ERROR.getCcdType(),
+                "Send to FTA Error",
+                description,
+                idamService.getIdamTokens(),
+                sscsCaseData -> {
+                    sscsCaseData.setHmctsDwpState("failedSending");
+                    log.info("Updated case v2 send to bulk print event -error for id {}", caseId);
+                }
+        );
     }
 
-    private void updateCaseToSentToDwp(Callback<SscsCaseData> sscsCaseDataCallback, SscsCaseData caseData,
+    private void updateCaseToSentToDwp(SscsCaseData caseData,
                                        BulkPrintInfo bulkPrintInfo) {
+        Long caseId = Long.valueOf(caseData.getCcdCaseId());
         if (bulkPrintInfo != null) {
-            if (State.READY_TO_LIST.getId().equals(caseData.getCreatedInGapsFrom())) {
-                caseData.setDwpState(UNREGISTERED);
-            }
-            caseData.setHmctsDwpState("sentToDwp");
-            caseData.setDateSentToDwp(LocalDate.now().toString());
-            caseData.setDwpDueDate(LocalDate.now().plusDays(getResponseDueDays(caseData)).toString());
-
-            ccdService.updateCase(caseData, Long.valueOf(caseData.getCcdCaseId()),
-                EventType.SENT_TO_DWP.getCcdType(), SENT_TO_FTA, bulkPrintInfo.getDesc(),
-                idamService.getIdamTokens());
-            if (bulkPrintInfo.isAllowedTypeForBulkPrint()) {
-                log.info("Case sent to fta for case id {} with returned value {}",
-                    sscsCaseDataCallback.getCaseDetails().getId(), bulkPrintInfo.getUuid());
-            }
+            log.info("About to update case v2 with send to bulk print event - dwp for id {}", caseId);
```

Comment:
I was advised on this pattern.

Reply:
parent method already has a log so don't think we need this additionally ?

Reply:
Should we log case id in the parent method so that logs can be correlated? https://github.com/hmcts/sscs-tribunals-case-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/callback/CallbackDispatcher.java#L41

Reply:
Added case id in CallbackDispatcher -> dispatchToHandlers method